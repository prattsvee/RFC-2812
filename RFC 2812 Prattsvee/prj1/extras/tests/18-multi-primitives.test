12_34
true
:some_long_atom_1234
false
833_4
