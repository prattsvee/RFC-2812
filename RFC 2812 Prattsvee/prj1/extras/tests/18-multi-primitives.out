[
  {
    "%k": "int",
    "%v": 1234
  },
  {
    "%k": "bool",
    "%v": true
  },
  {
    "%k": "atom",
    "%v": ":some_long_atom_1234"
  },
  {
    "%k": "bool",
    "%v": false
  },
  {
    "%k": "int",
    "%v": 8334
  }
]
